Casks/monofocus.rb

```diff
@@ -0,0 +1,18 @@
+cask "monofocus" do
+  version "1.0.beta35"
+  sha256 :no_check
+
+  url "https://updates.monofocus.app/MonoFocus.latest.dmg"
+  name "MonoFocus"
+  desc "Keep all tasks from your todo apps on your menu bar"
+  homepage "https://monofocus.app/"
+
+  livecheck do
+    url :url
+    strategy :extract_plist
```

Comment:
```suggestion
    regex(/^v?(\d+(?:\.\d+)+\S*)/i)
    strategy :extract_plist do |items, regex|
      items["com.zbudniewek.WorkingOn"]&.short_version&.scan(regex)&.map { |match| match[0] }
    end
```

Looks like we'll need to extract the version from the `short_version` text (e.g., `1.0.beta35 (32cc9073)`).